SportsHub/SportsHub.Application/Services/IArticleService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SportsHub.Domain.Models;
+
+namespace SportsHub.AppService.Services
+{
+    public interface IArticleService
+    {
+        Task<Article?> GetByTitleAsync(string title);
```

Comment:
is the Title unique? I don`t see that configuration in the DB

Reply:
Article is unique now.